sample-apps/aws-xray-sdk-go-sample/README.md

```diff
@@ -0,0 +1,23 @@
+# SampleApp for AWS X-Ray Go SDK
+
+This repository contains sample app to show the tracing use case of aws-xray-sdk-go. The SampleApp contains example of tracing aws sdk calls like list all s3 buckets. Moreover, it contains tracing upstream HTTP request. 
+
+## Prerequirements
+
+* Should have XRay daemon or AOC with xray receiver installed, and running in order to see traces on the AWS XRay console
+
+## Requst route path
+
+This application contains 2 paths
+```
+/aws-sdk-call
+/outgoing-http-call
+```
+
+## Opening Issues
```

Comment:
edit this part (since this is not aws-samples repo).

Reply:
OK